### PR TITLE
[IMP] mass_mailing: do not load images on add sidebar and commitChanges

### PR DESCRIPTION
**Pull request by Zynton**
Images in mass_mailing are not lazy loaded, which is deliberate for the mailing itself and its conversion, but when we load the sidebar we add the HTML to all the snippets and hide it so if these images are not lazy loaded, they all get loaded along with the sidebar, which is slow and unnecessary.

Additionally, failing to remove the sidebar from the conversion iframe made these images load again when committing changes.

This loads the images only once we add them to the mailing, which is when we remove the lazy loading attribute.

task-3323894

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/mass_mailing/static/src/js/mass_mailing_html_field.js [hunk 1]
```diff
@@ -120,6 +120,7 @@ export class MassMailingHtmlField extends HtmlField {
             const clonedBody = clonedHtmlNode.querySelector('body');
             const clonedIframeTarget = clonedHtmlNode.querySelector('#iframe_target');
             clonedBody.replaceChildren(clonedIframeTarget);
+            clonedBody.querySelector('.iframe-utils-zone').remove();
```
Comment (Zinston) [inline addons/mass_mailing/static/src/js/mass_mailing_html_field.js:123]: TODO: move to separate commit.

**Comment by dmo-odoo** [comment]
@robodoo rebase-ff r+
